website/docs/r/serviceendpoint_externaltfs.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_serviceendpoint_externaltfs"
+description: |-
+  Manages an Azure Repos/Team Foundation Server service endpoint within Azure DevOps organization.
+---
+
+# azuredevops_serviceendpoint_externaltfs
+
+Manages an Azure Repos/Team Foundation Server service endpoint within Azure DevOps.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "example" {
+  name               = "Example Project"
+  visibility         = "private"
+  version_control    = "Git"
+  work_item_template = "Agile"
+  description        = "Managed by Terraform"
+}
+
+resource "azuredevops_serviceendpoint_externaltfs" "example" {
+  project_id            = azuredevops_project.example.id
+  service_endpoint_name = "Example External TFS Name"
+  url                   = "https://dev.azure.com/myorganization"
+  description           = "Managed by Terraform"
+
+  auth_personal {
+    # Also can be set with AZDO_PERSONAL_ACCESS_TOKEN environment variable
+    personal_access_token = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `project_id` - (Required) The ID of the project.
+- `service_endpoint_name` - (Required) The Service Endpoint name.
+- `url` - (Required) Azure DevOps Organization or TFS Project Collection Url.
+- `description` - (Optional) The Service Endpoint description. Defaults to `Managed by Terraform`.
+- `auth_personal` - (Optional) An `auth_personal` block as documented below. Allows connecting using a personal access token.
```

Comment:
`auth_personal` is a required parameter not optional

Reply:
Updated.